Client/Frontend/Widgets/Menu/MenuViewController.swift

```diff
@@ -186,6 +186,20 @@ extension MenuViewController: MenuItemDelegate {
         performMenuAction(menuItem.action, withAnimation: animation, onView: menuItemCell.menuImageView)
     }
 
+    func menuView(menuView: MenuView, didLongPressItemAtIndexPath indexPath: NSIndexPath) {
```

Comment:
I was worried about using `selectedIcon`, when perhaps `selectedIcon` may not be appropriate for the `secondaryAction`. I was most concerned about using animations.

Will refactor.